traffic_ops/testing/api/v4/parameters_test.go

```diff
@@ -157,7 +176,31 @@ func UpdateTestParameters(t *testing.T) {
 	if respParameter.Value != expectedParameterValue {
 		t.Errorf("results do not match actual: %s, expected: %s", respParameter.Value, expectedParameterValue)
 	}
+}
 
+func UpdateParametersEmptyConfigFile(t *testing.T) {
+	if len(testData.Parameters) > 0 {
+		firstParameter := testData.Parameters[0]
+		// Retrieve the Parameter by name so we can get the id for the Update
+		opts := client.NewRequestOptions()
+		opts.QueryParameters.Set("name", url.QueryEscape(firstParameter.Name))
+		resp, _, err := TOSession.GetParameters(opts)
+		if err != nil {
+			t.Errorf("cannot GET Parameter by name: %v - %v", firstParameter.Name, err)
+		}
+		if len(resp.Response) > 0 {
```

Comment:
This test will silently pass if the test cannot proceed because Traffic Ops is returning improper responses and/or has no Parameters stored to use in testing.

Reply:
Added failure checks